web_external/js/views/widgets/WMSFeatureInfoWidget.js

```diff
@@ -5,7 +5,7 @@ minerva.views.WmsFeatureInfoWidget = minerva.View.extend({
 
         function getActiveWmsLayers() {
             return _.chain(that.parentView.collection.models)
-                .filter(function (set) { return set.get('displayed') && set.getDatasetType() !== 'geojson'; })
+                .filter(function (set) { return set.get('displayed') && set.get('visible') && set.getDatasetType() !== 'geojson' && set.getDatasetType() !== 'geojson-timeseries' })
```

Comment:
This (and two other places below) throws a linting test for a missing semicolon.